multi-node/aws/pkg/cluster/cloud_config_controller.go

```diff
@@ -14,11 +14,31 @@ coreos:
     initial-advertise-peer-urls: http://$private_ipv4:2380
     listen-client-urls: http://0.0.0.0:2379
     listen-peer-urls: http://0.0.0.0:2380
-    initial-cluster: controller=http://$private_ipv4:2380
 
   units:
+  - name: etcd-peers.service
```

Comment:
Were you able to explore any other options than this? Is there no way to assign static IPs at deploy time and have a static `initial-cluster` option?